.github/workflows/rerun-compare-chromium-versions.yml

```diff
@@ -0,0 +1,47 @@
+name: Rerun Compare Chromium versions
+
+on:
+  pull_request:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+.x'
+    paths:
+      - package.json
+    types:
+      - closed
```

Comment:
I had some issues with this event in the brave-variations repo.

https://github.com/orgs/community/discussions/26657

> The workaround is to use a `pull_request_target` trigger as those do not require a merge commit but run the workflow from the target branch. But take note of the warnings in the docs below.

Reply:
Is there a public issue tracking this from their side?